src/config/index.ts

```diff
@@ -326,6 +326,12 @@ export const BaseSettingsDefinition = {
     default: 90_000,
     validate: validator.integer({ min: 1000, max: 180_000 }),
   },
+  RATE_LIMITING_STRATEGY: {
+    description: 'The rate limiting strategy to use for outbound requests',
+    type: 'enum',
+    options: ['counting', 'fixed'],
+    default: 'fixed',
```

Comment:
This is something _additional_ documentation would help fix, but is there any way we can get more descriptive names of the strategies? I'd have no idea what the difference is between "counting" and "fixed" if I were to configure this.

Reply:
I agree that it's not super clear, but I couldn't think of any alternatives, any suggestions are welcome

Reply:
It doesn't necessarily describe the strategy and more so the behavior of requests but what if we do `burst` and `fixed-interval`? I think keeping these names is ok though if have a small explanation for each in documentation

Reply:
I think those are better, I'll rename and push a new commit here